deployments/engine/templates/deployment.yaml

```diff
@@ -69,6 +73,12 @@ spec:
         - name: config
           mountPath: /etc/config
           readOnly: true
+        - name: tmp
```

Comment:
Could you add some comment to explain why this change is needed? My understanding is that this is to load the models in memory. If so, I think this should be fine, but we just need to be a bit careful to make sure if we have sufficient memory when multiple models are loaded (this only apply to Ollama, though).

Reply:
Ah nvm. I misunderstood the behavior of `emptyDir`.